src/services/notifications/index.ts

```diff
@@ -191,13 +194,59 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     }
   }
 
+  const sendBalance = async (account: Account): Promise<void> => {
+    const balanceSats = await getBalanceForWallet({
+      walletId: account.defaultWalletId,
+      logger,
+    })
+    if (balanceSats instanceof Error) throw balanceSats
+
+    // Add commas to balancesats
```

Comment:
Are both these comments above and below 205 necessary? Can we encode the comment in the variable name:
```
const balanceSatsAsFormattedString = ...
```

Reply:
at least the comment is useful to me. I would need to know otherwise that `toLocalString("en")` will add the comms the `balanceSats` to understand this line, and this is not really something we use a lot in the codebase